release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -549,6 +549,12 @@ With this release, the control plane supports TLS 1.3. You can now use the `Mode
 
 For more information, see xref:../security/tls-security-profiles.adoc#tls-profiles-kubernetes-configuring_tls-security-profiles[Configuring the TLS security profile for the control plane].
 
+[id="ocp-release-notes-eso-operator_{context}"]
+==== External Secrets Operator for Red Hat OpenShift (Technology Preview)
+With this release, you can use the {external-secrets-operator}, available as a Technology Preview.
```

Comment:
A 1-2 liner summary of what this operator does is required for the user to understand the operator.